doc/source/Parameters.rst

```diff
@@ -45,6 +45,30 @@ For all on/off integer flags, 0 is off and 1 is on.
    and advect baryon fields for each of the species used by that
    particular option.
 
+.. c:var:: int dust_chemistry
+
+   Flag to control additional dust cooling and chemistry processes.
+   Default: 0.
+
+   - 0: no dust-related processes included.
+   - 1: adds the following processes:
+
+        #. photo-electric heating (sets :c:data:`photoelectric_heating` to 2).
+        #. cooling from electron recombination onto dust (equation 9 from
+           `Wolfire et al. 1995
+           <https://ui.adsabs.harvard.edu/abs/1995ApJ...443..152W/abstract>`__).
+           Both the photo-electric heating and recombination cooling are scaled
+           by the value of the :c:data:`interstellar_radiation_field`.
+        #. H\ :sub:`2`\  formation on dust (sets :c:data:`h2_on_dust` to 1
+           if :c:data:`primordial_chemistry` > 1).
+
+        Setting :c:data:`dust_chemistry` greater than 0 requires
+        :c:data:`metal_cooling` to be enabled.
+
+.. note:: Other values for :c:data:`photoelectric_heating` may also be used
+   in conjunction with setting the :c:data:`dust_chemistry` parameter. It will
+   only be changed to 2 if set to its default value of 0 first.
```

Comment:
Ok, I can do this. At some point, it would probably be good to come up with a policy for how granular we want parameters to be. I could even see wanting to have a parameter for every term, but maybe that could wait until the Fortran is converted to C so we don't have to pass everything as a function argument. I'll do this one as you suggest, though.